### ISSUE TITLE
dynamic visualisation tool has no graphic

### ISSUE THREAD

**Issue by syloktools** [MISP/MISP#3225]
### Work environment

| Questions                 | Answers
|---------------------------|--------------------
| Type of issue             | Bug
| OS version (server)       | ubuntu
| OS version (client)       | 10
| PHP version               | 7.0
| MISP version / git hash   | 2.4.XX, hash of the commit
| Browser                   | If applicable

### Expected behavior
See graphic like:
![image](https://user-images.githubusercontent.com/20773170/39650245-662eb102-4fb5-11e8-808b-1cb833b0bbf6.png)

### Actual behavior
Graphic looks like:
![image](https://user-images.githubusercontent.com/20773170/39650272-8112b892-4fb5-11e8-8714-773b75d126ec.png)

**Comment by adulau** [comment]
Did you reload the page? and if yes, could you give us the output of the JavaScript debug console. Thanks a lot.

**Comment by syloktools** [comment]
![image](https://user-images.githubusercontent.com/20773170/39714818-358e1466-51f9-11e8-8f07-24f1d3cdf8f7.png)

**Comment by syloktools** [comment]
```
 Uncaught TypeError: Cannot read property 'length' of undefined
    at Object.success (event-distribution-graph.js:236)
    at fire (jquery.js:3187)
    at Object.fireWith [as resolveWith] (jquery.js:3317)
    at done (jquery.js:8757)
    at XMLHttpRequest.<anonymous> (jquery.js:9123)
```

**Comment by mokaddem** [comment]
Thanks for the report, should be fixed in f1cefb350340337f4f40b6290a05b349330141b8.

**Comment by syloktools** [comment]
Works after https://github.com/MISP/MISP/commit/f1cefb350340337f4f40b6290a05b349330141b8
